src/jaxsim/rbda/contacts/relaxed_rigid.py

```diff
@@ -243,25 +243,26 @@ def compute_contact_forces(
                 Optional `(n_joints,)` vector of joint forces.
 
         Returns:
-            A tuple containing the contact forces.
+            A tuple containing as first element the computed contact forces.
         """
 
         # Initialize the model and data this contact model is operating on.
         # This will raise an exception if either the contact model or the
         # contact parameters are not compatible.
         model, data = self.initialize_model_and_data(model=model, data=data)
+        assert isinstance(data.contacts_params, RelaxedRigidContactsParams)
 
-        link_forces = (
-            link_forces
+        link_forces = jnp.atleast_2d(
+            jnp.array(link_forces, dtype=float).squeeze()
             if link_forces is not None
             else jnp.zeros((model.number_of_links(), 6))
-        )
+        ).astype(float)
 
-        joint_force_references = (
-            joint_force_references
+        joint_force_references = jnp.atleast_1d(
+            jnp.array(joint_force_references, dtype=float).squeeze()
```

Comment:
Same as above, probably a bit redundant 
```suggestion
            jnp.array(joint_force_references).squeeze()
```